cirq-core/cirq/protocols/unitary_protocol_test.py

```diff
@@ -209,7 +209,6 @@ def _test_gate_that_allocates_qubits(gate):
 def test_decompose_gate_that_allocates_clean_qubits(
     theta: float, phase_state: int, target_bitsize: int, ancilla_bitsize: int
 ):
-
```

Comment:
Revert this file please, since these whitespace changes are not relevant to the PR.